test/integration/scheduler/predicates_test.go

```diff
@@ -1041,18 +1042,45 @@ var (
 func TestUnschedulablePodBecomesSchedulable(t *testing.T) {
 	tests := []struct {
 		name   string
-		init   func(kubernetes.Interface) error
+		init   func(kubernetes.Interface, string) error
 		pod    *pausePodConfig
-		update func(kubernetes.Interface) error
+		update func(kubernetes.Interface, string) error
 	}{
 		{
 			name: "node gets added",
 			pod: &pausePodConfig{
-				Name: "pod-1",
+				Name: "pod-" + uuid.New().String(),
```

Comment:
This also doesn't need to be unique

Reply:
Got it, the namespace is unique anyway 👍  I will revert it.

Reply:
the pod not being unique is fine.